Sources/SwiftFusion/Core/MathUtil.swift

```diff
@@ -4,16 +4,16 @@
 
 import TensorFlow
 
-public func pinv(_ m: Tensor<Double>) -> Tensor<Double> {
+public func pinv<Scalar: TensorFlowFloatingPoint>(_ m: Tensor<Scalar>) -> Tensor<Scalar> {
```

Comment:
Document, especially about what input is actually handled, and what output dimensions are